plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -18,6 +18,12 @@ NVENC.Preset.ll="Low-Latency"
 NVENC.Preset.llhq="Low-Latency High Quality"
 NVENC.Preset.llhp="Low-Latency High Performance"
 NVENC.Level="Level"
+NVENC.SpatialStrength="Spatial AQ Strength"
```

Comment:
You should probably use "NVENC.SpatialAQ.Strength" for the string name and place it below "NVENC.SpatialAQ".